src/app/models/Property.ts

```diff
@@ -0,0 +1,42 @@
+import { Client } from "./Client";
+import { Coordinates } from "./Coordinates";
+import { Address } from "./Address";
+
+export interface Property {
+  id?: number;
+  price?: number;
+  lost_size?: number;
+  image_others?: string[];
+  address?: Address;
+  coordinates?: Coordinates;
+  created_at?: string;
+  updated_at?: string;
+  title?: string;
+  property_type?: string;
+  description?: string;
+  list_date?: string;
+  is_published?: boolean;
+  is_sold?: boolean;
+  sold_at?: string;
+  bedrooms?: number;
+  bathrooms?: number;
+  garages?: number;
+  image_main?: string;
+  video?: string;
+  view_count?: number;
+  last_viewed?: string;
```

Comment:
@Genza999, please use cameCase for variable naming, Its customary JS convention.

Reply:
Hey @CryceTruly I can't because the structure of the data from the backend is in this format. Kindly elaborate on this since these are interfaces.

Reply:
Hey @CryceTruly I have changed it tp CamelCase thanks